README.md

```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
 }--->
 ```
 
+#### How to style examples
+
+You can’t reference external stylesheets when creating AMP pages. All styles must live in the head of the document using the tag `<style amp-custom>`.
+For AMPByExample, we are using templates, so that we can manage better the CSS, as suggested [here](https://www.ampproject.org/docs/guides/responsive/style_pages.html).
+After you write the example and the page is rendered by gulp, the CSS classes will be added to the AMP page from the existing CSS files.
+If you copy the example directly from the code section, you will not get the style that you can see in the demo section. We try to limit the CSS that we add to the examples only adding the relevant one for the specific AMP component that we are testing; all the other required CSS will be added after the template is rendered.
```

Comment:
suggestion: rephrase first sentence to

"Please note: if you copy code from a sample's code section, you will not get the style that you can see in the preview section."